playwright/tests/parallel-4/pageskin.spec.ts

```diff
@@ -57,7 +57,7 @@ test.describe('pageskin on uk front', () => {
 	 * The test for tablet always passes locallly but always fails on CI
 	 */
 	small.forEach(({ breakpoint, width, height }) => {
-		test(`Test pageskin front on ${breakpoint} should NOT display the pageskin and NOT use single request mode`, async ({
+		test.skip(`Test pageskin front on ${breakpoint} should NOT display the pageskin and NOT use single request mode`, async ({
```

Comment:
Could we just skip tablet if that's the problem one. No worries if just simpler to skip both

Reply:
Yep, absolutely. I thought it was both tablet and mobile but you are correct. I will return early with that comment above for tablet